x-pack/plugins/lists/server/routes/delete_list_route.ts

```diff
@@ -9,9 +9,18 @@ import { IRouter } from 'kibana/server';
 import { LIST_URL } from '../../common/constants';
 import { buildRouteValidation, buildSiemResponse, transformError } from '../siem_server_deps';
 import { validate } from '../../common/shared_imports';
-import { deleteListSchema, listSchema } from '../../common/schemas';
+import {
+  EntriesArray,
+  ExceptionListItemSchema,
+  FoundExceptionListSchema,
+  deleteListSchema,
+  exceptionListItemSchema,
+  listSchema,
+} from '../../common/schemas';
+import { getSavedObjectType } from '../services/exception_lists/utils';
+import { ExceptionListClient } from '../services/exception_lists/exception_list_client';
 
-import { getListClient } from '.';
+import { getExceptionListClient, getListClient } from '.';
 
 export const deleteListRoute = (router: IRouter): void => {
```

Comment:
This whole function is pretty long and hard to read, so could probably benefit from a little refactoring.

Reply:
You should've saw it before I abstracted out `getReferencedExceptionLists` and `deleteExceptionListItem` 😅. But yeah, will see if I can abstract out a few more functions to help with complexity/readability. 👍